install_dependencies_ubuntu_14_04.sh

```diff
@@ -3,4 +3,3 @@ set -x
 # this is run on the VM to install docker and all dependencies of kastenwesen.py
 sudo apt-get -q update
 sudo apt-get -y -q install docker.io apparmor-profiles python-pip aha
-sudo pip install -r requirements.txt
```

Comment:
This is a significant change and breaks the way that install_dependencies.sh worked before. Now it no longer installs all required dependencies.